torch/_inductor/codecache.py

```diff
@@ -472,7 +472,17 @@ def write_atomic(
     write_mode = "w" if isinstance(content, str) else "wb"
     with tmp_path.open(write_mode, encoding="utf-8" if encode_utf_8 else None) as f:
         f.write(content)
-    tmp_path.rename(path)
+    try:
+        tmp_path.rename(target=path)
+    except FileExistsError as e_file_exist:
+        if not _IS_WINDOWS:
+            raise
```

Comment:
See https://docs.python.org/3/tutorial/errors.html#handling-exceptions - as far as I understand, if one want to preserve stacktrace, right way to re-raise is just calling `raise` from the except handler (or `raise e from None`)